parsers/to-react-native/to-react-native.parser.ts

```diff
@@ -0,0 +1,95 @@
+import { BitmapToken, IToken, Token, VectorToken } from '../../types';
+import prettier from 'prettier';
+import * as TokensClass from './tokens';
+import { LibsType } from '../global-libs';
+import Template from '../../libs/template';
+import { camelCase } from 'lodash';
+
+export type InputDataType = Array<
+  Pick<IToken, 'id' | 'name' | 'value' | 'type'> & Record<string, any>
+>;
+export type OutputDataType = string;
+export type ColorsFormat =
+  | 'rgb'
+  | 'prgb'
+  | 'hex'
+  | 'hex6'
+  | 'hex3'
+  | 'hex4'
+  | 'hex8'
+  | 'name'
+  | 'hsl'
+  | 'hsv';
+
+export type FormatTokenType = Partial<{}>;
+
+export type OptionsType =
+  | Partial<{
+      assetsFolderPath?: string | { vector?: string; bitmap?: string };
+      colorFormat: ColorsFormat;
+      header: string;
+      objectName: string;
+      prettierConfig: prettier.Options;
+    }>
+  | undefined;
+
+export default async function (
+  tokens: InputDataType,
+  options: OptionsType,
+  { _ }: Pick<LibsType, '_'>,
+): Promise<OutputDataType> {
+  try {
+    const objectName = options?.objectName || 'theme';
+    const pattern =
+      '{{name}}{{#dimension}}@{{dimension}}x{{/dimension}}{{#format}}.{{format}}{{/format}}';
+    const tokensGroupByType = _.groupBy(tokens, 'type');
+    const template = new Template(pattern);
+    let imports = '';
+    const styles = Object.keys(tokensGroupByType)
+      .filter(type => type !== 'shadow') // Only support depth for now
```

Comment:
It will create an empty `shadows: {}` entry in the output, which is undesired I think.

Reply:
You're right. 
I suggest omitting all types that include empty content instead of filter only on `shadow`. 
To do that you could remove the filter and wrap the concatenation of the result in a condition like:
```ts
if (content) {
  result += `${_.camelCase(type)}: {${content}},`;
}
```